src/features/registry.js

```diff
@@ -12,6 +14,9 @@ systemJSPrototype.get = function (id) {
 };
 
 systemJSPrototype.set = function (id, module) {
+  if (!process.env.SYSTEM_PRODUCTION && !resolveIfNotPlainOrUrl(id, baseUrl)) {
+    console.warn(Error(errMsg(53, 'Invalid module id - ' + id + ' must be url')));
```

Comment:
```suggestion
    console.warn(Error(errMsg(53, '"' + id + '" is not a valid URL to set in the module registry')));
```

I don't think a slightly longer message is a problem for these dev builds.

If the `Error` is for the stack, I think we still get a stack with warnings?

Reply:
Would still be interested to hear your thoughts on this question.